templates/CRM/common/civicrm.settings.php.template

```diff
@@ -502,6 +502,13 @@ if (CIVICRM_UF === 'UnitTests') {
 //   define('CIVICRM_LOG_ROTATESIZE', 0 );
 // }
 
+/**
+ * Determine if we should connect to the database over SSL
+ */
+// if (!defined('CIVICRM_DB_USE_SSL')) {
+//   define('CIVICRM_DB_USE_SSL', FALSE);
+// }
+
```

Comment:
I haven't tried running this yet but just commenting I'd put this block up near where the DSN's are defined.